src/utils/selectors/index.ts

```diff
@@ -18,9 +21,11 @@ export const typedStore = `MethodDefinition[kind='constructor'] Identifier>TSTyp
 
 export const ngModuleDecorator = `ClassDeclaration > Decorator > CallExpression[callee.name='NgModule']`
 
-export const ngModuleProviders = `${ngModuleDecorator} ObjectExpression Property[key.name='providers'] > ArrayExpression Identifier`
+export const ngModuleProviders =
+  `${ngModuleDecorator} ObjectExpression Property[key.name='providers'] > ArrayExpression Identifier` as const
 
-export const ngModuleImports = `${ngModuleDecorator} ObjectExpression Property[key.name='imports'] > ArrayExpression CallExpression[callee.object.name='EffectsModule'][callee.property.name=/forRoot|forFeature/] ArrayExpression > Identifier`
+export const ngModuleImports =
+  `${ngModuleDecorator} ObjectExpression Property[key.name='imports'] > ArrayExpression CallExpression[callee.object.name='EffectsModule'][callee.property.name=/forRoot|forFeature/] ArrayExpression > Identifier` as const
 
 export const actionDispatch = (storeName: string) =>
   `ExpressionStatement > CallExpression:matches([callee.object.name='${storeName}'][callee.property.name='dispatch'], [callee.object.object.type='ThisExpression'][callee.object.property.name='${storeName}'][callee.property.name='dispatch'])`
```

Comment:
Although we could do for functions/selectors that deal with `storeName`, it wouldn't be used much since the storeName is fully dynamic.

Not to mention that, honestly, it would be quite verbose like this:

```suggestion
export const actionDispatch = <TStore extends string>(
  storeName: TStore,
): `ExpressionStatement > CallExpression:matches([callee.object.name='${TStore}'][callee.property.name='dispatch'], [callee.object.object.type='ThisExpression'][callee.object.property.name='${TStore}'][callee.property.name='dispatch'])` =>
  `ExpressionStatement > CallExpression:matches([callee.object.name='${storeName}'][callee.property.name='dispatch'], [callee.object.object.type='ThisExpression'][callee.object.property.name='${storeName}'][callee.property.name='dispatch'])`
```

Reply:
Seems like a good call 😅

Reply:
I just noticed that we could add `as const` without the need of explicit return type, so instead of `string` we get this:

<img width="830" alt="as const" src="https://user-images.githubusercontent.com/11965907/132265273-543630e0-0154-48a3-9d05-2206f5667245.png">